addons/l10n_sa/models/account_move.py

```diff
@@ -2,20 +2,24 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 import base64
 
-from odoo import api, fields, models, _
+from odoo import _, api, fields, models
 from odoo.exceptions import UserError
-from odoo.tools import float_repr
+from odoo.tools import float_repr, format_datetime
 
 
 class AccountMove(models.Model):
     _inherit = 'account.move'
 
-    l10n_sa_delivery_date = fields.Date(string='Delivery Date', default=fields.Date.context_today, copy=False,
+    l10n_sa_delivery_date = fields.Date(string='Supply Date', default=fields.Date.context_today, copy=False,
```

Comment:
This should be part of the pot and translated in the .po in Arabic.
(I know, the previous word was not but we should do better :smile: )

Same for the help, etc... below